spec/models/logstash_queries/count_query_spec.rb

```diff
@@ -1,10 +1,25 @@
 require 'spec_helper'
 
-describe CountQuery, "#body" do
+describe CountQuery do
   let(:query) { CountQuery.new('affiliate_name') }
+  let(:expected_body) do
+    {
```

Comment:
Note: I'm using string keys here with double quotes to reflect the actual JSON we send to Elasticsearch. This also simplifies investigation and debugging, because queries in this format can be pasted directly into Kibana.